### ISSUE TITLE
Focused Launch: Plans Grid unavailable Free plan copy & styles in Plan details view

### ISSUE THREAD

**Issue by StefanNieuwenhuis** [Automattic/wp-calypso#47788]
Currently used in Focused Launch - Plans Details view, the version of `<PlansTable/>` should have specific styling & copy for unavailable plans but Production doesn't match the specs specified in Figma. See the screenshots below for comparison.

#### Acceptance Criteria

* [ ] The cross icon's color for the unavailable plan is orange (`--studio-orange-40`).
* [ ] The unavailable domain copy is crossed through.
* [ ] The disabled button's copy is: `Unavailable with domain`
* [ ] The orange copy is: `[domain] is not included`

#### Additional information
Button styles are tackled separately in #47740.

#### What I expected

![Screenshot 2020-11-27 at 13 27 52](https://user-images.githubusercontent.com/1083581/100449514-5b5cff80-30b4-11eb-97fa-b6984796e086.png)

#### Current situation on Production
<img width="304" alt="Screenshot 2020-11-26 at 11 24 16" src="https://user-images.githubusercontent.com/12104589/100339279-f7660880-2fd9-11eb-86d4-d14cdc97422c.png">

**Comment by ciampo** [comment]
Hey @ollierozdarz, in a similar fashion to [this other comment](https://github.com/Automattic/wp-calypso/issues/47740#issuecomment-734225642), can you confirm if those changes should apply to the Plans Grid only in the case of Focused Launch, or if they should be applied to all instances of the Plans Grid?

**Comment by ollierozdarz** [comment]
Hey @ciampo, the intention of all the focused launch flow styles is to be contained to this flow. We're using some new styles scattered throughout this task/project and I'd prefer to keep them contained here instead of globally rolling them out to older screens where we don't know how they'll look/work once incorporated 🙂 

If we have to use them globally it _should_ be fine, but not ideal.

**Comment by ciampo** [comment]
Hey @ollierozdarz, I see your point. Together with the team, we've decided that those changes will also be reflected in the New Onboarding (gutenboarding) and "Step By Step" Launch flows (the launch flow for sites created with gutenboarding), as they share the components with Focused Launch. This greatly simplifies the implementation of those tweaks, and will help us maintaining those components in the future.

This should be a nice happy medium — other (older) screens and flows won't be affected, since they don't use this version of the plans grid and the domain picker

**Comment by StefanNieuwenhuis** [comment]
Heya @ollierozdarz!
While comparing Figma and the current situation on Production, I noticed that the unavailable domains copy (the text in orange, after the cross icon) is different. Which one should we use?

Thank you so much! 

#### Variant 1 (Figma)

<img width="381" alt="Screenshot 2021-01-08 at 13 01 02" src="https://user-images.githubusercontent.com/12104589/104013634-054ea200-51b2-11eb-85dc-d9aef44d2836.png">

#### Variant 2 (Production)

<img width="222" alt="Screenshot 2021-01-08 at 13 01 32" src="https://user-images.githubusercontent.com/12104589/104013595-f536c280-51b1-11eb-8129-313ec2262bdc.png">

**Comment by StefanNieuwenhuis** [comment]
Heya @ollierozdarz!
While comparing Figma and Focused Launch Plans Table that's currently on Production, I noticed that the button's copy is different. 

And there's also a third variant, which is found in the Step-by-Step Plans Accordion.

Which copy should we use?

#### Variant 1 (Figma)

<img width="413" alt="Screenshot 2021-01-08 at 13 07 15" src="https://user-images.githubusercontent.com/12104589/104013921-73936480-51b2-11eb-86e1-c57cba3b127e.png">

#### Variant 2 (Production - Focused Launch Plans Table)
<img width="231" alt="Screenshot 2021-01-08 at 13 07 08" src="https://user-images.githubusercontent.com/12104589/104013951-8017bd00-51b2-11eb-81e0-028dd26e9b7b.png">

#### Variant 3 (Production - Step-by-Step Plans Accordion)

<img width="309" alt="Screenshot 2021-01-08 at 13 09 16" src="https://user-images.githubusercontent.com/12104589/104014056-ba815a00-51b2-11eb-98b9-a916a2dafbf0.png">

**Comment by ciampo** [comment]
Let's also keep in mind that some languages are more verbose than english, and therefore the copy may not fit on one line even if we go for the shorter "Unavailable with domain" alternative (particularly relevant for the table variant used in Focused Launch flow)

**Comment by ollierozdarz** [comment]
Hi @StefanNieuwenhuis and @ciampo. 

I was hoping to use the '_Unavailable with domain_' text as it explains the situation better than the current '_Select free_', i.e. We're telling the user that this plan is unavailable to select because they've selected a premium domain, rather than providing no context.

For the text in Orange (domain X is not available), this may have just been an oversight on my part. I'm happy to use the current production language here 🙂

Thanks for checking!

**Comment by ciampo** [comment]
Hey @ollierozdarz, thanks for your reply!

For the button in the disabled _Free Plan_ card, we'll use  ~`Unavailable with domain`~  (across both table and accordion variants):

<img width="413" alt="Screenshot 2021-01-08 at 13 07 15" src="https://user-images.githubusercontent.com/12104589/104013921-73936480-51b2-11eb-86e1-c57cba3b127e.png">

For the text in orange, we can go with what's currently in production ~`X Not available with custom domain`~ — although that may be a bit of a repetition of the text in the disabled button, which is why I believe in the Focused Launch designs it was changed to ~`X [domain] is not included`~.

Just wanted to flag this potential repetition, in case you missed — we're happy to go for whichever you believe is better!

**Comment by StefanNieuwenhuis** [comment]
#### TL;DR
* Button's copy will be changed to: `Unavailable with domain`
* Orange text will be changed to: `[domain] is not included`

-----

Thanks for your reply, Ollie! 

> Let's also keep in mind that some languages are more verbose than English, and therefore the copy may not fit on one line

@ciampo I've checked the new copy in desktop mode and the copy breaks in both English as more verbose languages but only on the `$break-huge` (i.e. screens `> 1440px`) breakpoint. Besides the fact that we cannot do much about this, it doesn't affect UX either so we can safely implement the proposed copy.

@ollierozdarz, we've decided to change the copy of the orange text to `[domain] is not included` and update all flows/variants (see https://github.com/Automattic/wp-calypso/issues/47788#issuecomment-734790889). We believe that the UX will benefit from this since we don't fill up screen estate with duplicates.

**Comment by ollierozdarz** [comment]
Nice work, @ciampo and @StefanNieuwenhuis 🙂 

Do we have any screenshots of how these changes look in the zoomed out UI (i.e. not just a screenshot of the button) in the focused launch flow and any other locations it'll be included?

**Comment by StefanNieuwenhuis** [comment]
@ollierozdarz thanks! Can you please confirm that these changes are correct so that we can publish them? 

**Good to know!** The orange buttons are due to the selected `wp-admin` theme and not caused by an unwanted side-effects of our code. 

> Do we have any screenshots of how these changes look in the zoomed out UI (i.e. not just a screenshot of the button) in the focused launch flow and any other locations it'll be included?

#### Focused Launch
<img width="1286" alt="Screenshot 2021-01-13 at 10 01 17" src="https://user-images.githubusercontent.com/12104589/104430030-82995e80-5586-11eb-89d6-111e2219052d.png">

<img width="376" alt="Screenshot 2021-01-13 at 10 02 13" src="https://user-images.githubusercontent.com/12104589/104430018-7f9e6e00-5586-11eb-838a-823190841f5f.png">

#### Gutenboarding
<img width="1466" alt="Screenshot 2021-01-13 at 10 25 08" src="https://user-images.githubusercontent.com/12104589/104432860-b9bd3f00-5589-11eb-994f-6070a6c6ac6b.png">

<img width="338" alt="Screenshot 2021-01-13 at 10 25 29" src="https://user-images.githubusercontent.com/12104589/104432855-b88c1200-5589-11eb-8c20-ca64f9b17a90.png">

#### Step-by-Step Launch
<img width="1709" alt="Screenshot 2021-01-13 at 10 50 37" src="https://user-images.githubusercontent.com/12104589/104435958-4b7a7b80-558d-11eb-889b-26f86be8e54b.png">

<img width="368" alt="Screenshot 2021-01-13 at 10 51 15" src="https://user-images.githubusercontent.com/12104589/104435950-4a494e80-558d-11eb-9e71-eca489838c60.png">

**Comment by ollierozdarz** [comment]
@StefanNieuwenhuis these language changes on the labels look good to me!
